lib/app/feature1/logic/bloc/test/test_bloc.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+part 'test_event.dart';
+part 'test_state.dart';
+
+class TestBloc extends Bloc<TestEvent, TestState> {
```

Comment:
Are we using Bloc or Cubit? We have to know since I’ll want us to be consistent.